ambry-router/src/main/java/com.github.ambry.router/DeleteOperation.java

```diff
@@ -124,6 +124,10 @@ private void fetchRequests(RequestRegistrationCallback<DeleteOperation> requestR
       RequestInfo requestInfo = new RequestInfo(hostname, port, deleteRequest);
       requestRegistrationCallback.registerRequestToSend(this, requestInfo);
       replicaIterator.remove();
+      if (RouterUtils.isRemoteReplica(routerConfig, replica)) {
+        logger.trace("Making request to a remote replica in", replica.getDataNodeId().getDatacenterName());
```

Comment:
Would be good to include request type. E.g., "Making Delete request to a remote replica in..."